python_modules/airline-demo/airline_demo/solids.py

```diff
@@ -612,11 +623,204 @@ def union_spark_data_frames(_info, left_data_frame, right_data_frame):
         cast(cast(depdelay as float) as integer) as departure_delay,
         origin,
         dest as destination
-    from q2_on_time_data
-    where origin='SFO'
+    from q2_sfo_outbound_flights
     ''',
     'table',
     table_name='average_sfo_outbound_avg_delays_by_destination',
+    inputs=[InputDefinition('q2_sfo_outbound_flights', dagster_type=SqlTableName)]
+)
+
+ticket_prices_with_average_delays = sql_solid(
```

Comment:
we should probably end up parameterizing this with a template

Reply:
totally, i think i want to reduce this machinery and the existing jinja-based stuff in the dagster_examples to a single reusable pattern